src/suvela/NumberArray.java

```diff
@@ -0,0 +1,33 @@
+package suvela;
+
+import java.util.Scanner;
+
+public class NumberArray {
+	int count = 0;
+
+	void findfrequency(int[] array, int num) {
+		boolean flag = false;
```

Comment:
I don't think so we need flag as variable here?
I don't see see any usage of flag variable.